src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
Follow the encapsulation principle. The `storage` map should be private to prevent direct access and modification from outside of the `Storage` class. Consider providing access through public methods.